modules/op-release-notes-1-7.adoc

```diff
@@ -27,7 +27,7 @@ The `preview` and `stable` channels will be deprecated and removed in a future r
 ** To add the annotations to a specific namespace, export the cluster task as a task to that namespace.
 // https://issues.redhat.com/browse/SRVKP-1514
 
-* With this update, the `when` expressions in a `Task` object are scoped to guard the tasks by default. To continue guarding the `Task` object and its dependent tasks, set the `scope-when-expressions-to-task` flag to `true`.
+* This update adds a new flag named `scope-when-expressions-to-task`. Before this update, the `when` expression would skip a `Task` object and its dependent tasks when certain conditions were not met. With this update, you can scope the `when` expression to guard the `Task` object only, not its dependent tasks. To enable this update, set the `scope-when-expressions-to-task` flag to true.
```

Comment:
Is 1.7 the first time the `scope-when-expressions-to-task` appear ? 🤔 
The rest is good, and the following should be added as well.

> The `scope-when-expressions-to-task`flag is deprecated and will be removed in a future release. As a best practice for {pipelines-shortname}, use`when`expressions scoped to the guarded`Task` only.

Reply:
@vdemeester, Removed the first sentence to avoid any ambiguity and kept the rest of the information as is. Please verify and confirm. Thanks